src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -238,7 +240,8 @@ export function GraphWrapper({
 				setSelectedRows(state.selectedRows);
 				break;
 			case DidChangeRefsVisibilityNotificationType:
-				setHiddenRefsById(state.hiddenRefs);
+				setExcludeRefsById(state.excludeRefs);
+				setIncludeOnlyRefsById(state.includeOnlyRefs);
```

Comment:
Should we include `excludeTypes` here? Or should be break them into separate notifications? I think combining them together works.

Reply:
We can probably combine them for ease of use, though Stashes technically aren't refs and we may introduce other non-ref types in the future.